packages/botonic-cli/src/commands/deploy.ts

```diff
@@ -350,10 +340,27 @@ Uploading...
       track('Deploy Botonic Provider Error', { error: e })
       console.log(colors.red(`There was an error getting the providers: ${e}`))
     }
+  }
+
+  async deploy() {
```

Comment:
🆒 isn't it a pleasure to read this kind of code? 😺

Reply:
for sure!!